app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -343,6 +349,19 @@ export class DashboardItemPage extends Component {
                     {formatMessage(messages.delete)}
                   </GhostButton>
                 )}
+                <GhostLink
```

Comment:
According to the story these buttons should be hidden and corresponding actions should be moved to the `actions` using `GhostMenuButton` (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/62532599-6b145d00-b84d-11e9-8c7a-ee0d75ca351a.png)

Reply:
Task description has been changed. Leave it as it is.